x-pack/plugins/apm/public/components/app/RumDashboard/CoreVitals/PaletteLegends.tsx

```diff
@@ -60,7 +82,13 @@ export function PaletteLegends({
             )}
             position="bottom"
           >
-            <PaletteLegend color={color}>{ranks?.[ind]}%</PaletteLegend>
+            <StyledSpan darkMode={darkMode}>
+              <PaletteLegend color={color}>
+                <EuiText size="xs">
+                  {labels[ind]} ({ranks?.[ind]}%)
```

Comment:
This should be a translated string.
```suggestion
<FormattedMessage
  id="xpack.apm.rum.coreVitals.paletteLegend.rankPercentage"
  defaultMessage="{labelsInd} ({ranksInd}%)"
  values={{
    labelsInd: labels[ind],
    ranksInd: ranks?.[ind],
  }}
/>
```